tests/component/test_stream_readers.py

```diff
@@ -562,49 +562,100 @@ def di_single_channel_multi_line_task(
     task: nidaqmx.Task, sim_6363_device: nidaqmx.system.Device
 ) -> nidaqmx.Task:
     task.di_channels.add_di_chan(
-        flatten_channel_string(sim_6363_device.di_lines[:2].name),
+        flatten_channel_string(sim_6363_device.di_lines[:4].name),
```

Comment:
FYI, I tried using this technique in another test and it doesn't really work right. It relies on the GIGO behavior described in https://github.com/ni/nidaqmx-python/issues/392

`di_lines[:4]` returns a single `DIChannel` object. Its `name` property should be a flattened channel string, but it's actually a Python list of channel names.